src/cws_pattern.js

```diff
@@ -51,6 +56,10 @@ function get_extensionID(url) {
     var match = cws_pattern.exec(url);
     if (match) return match[1];
     match = cws_download_pattern.exec(url);
+    if (match) return match && match[1];
```

Comment:
Remove these changes. This function is only used to detect extensionIDs of Chrome extensions, for use with the Chrome Web Store.

Using this logic with mea_ would result in all mea extension IDs being combined with the Chrome Web Store, which is not desired.

Reply:
Got it. Undid these changes.